cypress/integration/Change Password Tests/changePassword.specs.js

```diff
@@ -0,0 +1,43 @@
+ /// <reference types="cypress" />
+ import {passwordTab} from  '../../utils/changePasswordTab'
+  describe("NeetoAuth change password functionality", () => {
+    
+  it("Changing current password and trying to valid it can't login anymore with old password",()=>{
```

Comment:
Please fix the formatting.
```suggestion
  it("Changing current password and trying to valid it can't login anymore with old password", () => {
```